### PR TITLE
Drop Python 3.7 support

### PR DESCRIPTION
**Pull request by slovdahl**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ repository = "https://github.com/hiboxsystems/marge-bot"
 keywords = ["merge request", "gitlab"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.8"
```
Comment (perlun) [inline pyproject.toml:12]: This means "3.8 or greater, but less than 4.0", right?

Comment (slovdahl) [reply pyproject.toml:12]: No idea :grin:

**Review by perlun — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (slovdahl) [reply pyproject.toml:12]: Seems about right.

https://python-poetry.org/docs/dependency-specification/

image

Comment (perlun) [reply pyproject.toml:12]: OK, kudos for confirming.

**Pull request merged**